SampleApp/SampleApp/MessageCenterViewController.xaml.cs

```diff
@@ -18,10 +17,22 @@ public MessageCenterViewController()
         protected override void OnAppearing()
         {
             var messages = Airship.Instance.InboxMessages;
+
+            Console.WriteLine("Inbox Messages:");
+
+            foreach (Message message in messages)
+            {
+                Console.WriteLine("---");
+                Console.WriteLine(message.Title);
+                Console.WriteLine(message.Unread);
+                Console.WriteLine(message.SentDate);
+            }
+            Console.WriteLine("---");
```

Comment:
was this for test purpose ?

Reply:
Yeah, the linker was stripping out some fields from the `Message` class when the linking mode was set to `Full`, and that was causing the message list to look blank/empty, even though there were clickable messages in the list. 🤷 

I added this so I could see what was going on and thought it might be helpful to leave, since it's in the sample code and could be helpful if anyone runs into similar issues with their linker setting.

Reply:
Oh right, I didn't notice 👍